storage-node-v2/src/services/webApi/app.ts

```diff
@@ -55,20 +43,16 @@ export async function createApp(
         resolver: OpenApiValidator.resolvers.modulePathResolver,
       },
       fileUploader: {
-        dest: uploadsDir,
+        dest: config.tempFileUploadingDir,
         // Busboy library settings
         limits: {
           // For multipart forms, the max number of file fields (Default: Infinity)
           files: 1,
           // For multipart forms, the max file size (in bytes) (Default: Infinity)
-          fileSize: maxFileSize,
-        },
-      },
-      validateSecurity: {
-        handlers: {
-          UploadAuth: validateUpload(api, account),
+          fileSize: config.maxFileSize,
         },
       },
+      validateSecurity: setupUploadingValidation(config.enableUploadingAuth, config.api, config.storageProviderAccount),
```

Comment:
That seems to contradict the library intention, probably, that could be the case that header is received along with the first part of the file. Since we disabling the token - I'd rather not debug it now. I will remove the auth validation from the spec and the token endpoint.

Reply:
Fixed here: https://github.com/Joystream/joystream/pull/2686/commits/bcb65423693a3e0f2831aa60271e06d7524cd7bf